package/PartSeg/common_backend/base_settings.py

```diff
@@ -24,31 +28,48 @@ class ImageSettings(QObject):
     """
 
     image_changed = Signal([Image], [int], [str])
+    image_spacing_changed = Signal()
     """:py:class:`Signal` ``([Image], [int], [str])`` emitted when image has changed"""
-    segmentation_changed = Signal(np.ndarray)
+    segmentation_changed = Signal(SegmentationInfo)
     """
     :py:class:`.Signal`
     emitted when segmentation has changed
     """
     segmentation_clean = Signal()
-    noise_remove_image_part_changed = Signal()
+    additional_layers_changed = Signal()
 
     def __init__(self):
         super().__init__()
         self._image: Optional[Image] = None
         self._image_path = ""
         self._image_spacing = 210, 70, 70
         self._segmentation_info = SegmentationInfo(None)
-        self._noise_removed = None
+        self._additional_layers = {}
+
+    @property
+    def full_segmentation(self):
+        raise AttributeError("full_segmentation not supported")
+
+    @full_segmentation.setter
+    def full_segmentation(self, val):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=5714517)